server/db/migrate/20221020053651_create_join_table_companies_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateJoinTableCompaniesUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_join_table :companies, :users do |t|
+      # t.index [:company_id, :user_id]
+      # t.index [:user_id, :company_id]
+      # t.index :company_id
```

Comment:
I can't quite tell, but I think you modified the schema.rb when you meant to be changing this file to change the fields of the companies model